joomla/joomla-cms
# [5.0] Joomla Dialog for Aricles, Categories batch and Finder Index with Statistic

### Summary of Changes

An implementation of Joomla Dialog for  for Aricles, Categories batch and Finder Index with Statistic.
Depends from #40150

~It works but not fully finished.~ 
~It still will be need some stilyng for batch modals and solution for 'onclose'~ :potato: https://github.com/joomla/joomla-cms/blob/0659dc263f9461af20ccab361de19796ca9aafbd/administrator/components/com_finder/src/View/Index/HtmlView.php#L183

### Testing Instructions

Apply patch then apply #40150, run `npm install`.
Check Article, Categories batch, and Finder Index wtih Statistic popups. Versions popup


### Actual result BEFORE applying this Pull Request
Works


### Expected result AFTER applying this Pull Request
Works


### Link to documentations
Please select:
- [x] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/184
